server/config/database_test.go

```diff
@@ -153,7 +153,6 @@ func TestDatabaseStoreNew(t *testing.T) {
 		defer ds.Close()
 
 		assert.Equal(t, *customConfigDefaults.ServiceSettings.SiteURL, *ds.Get().ServiceSettings.SiteURL)
-		assert.Equal(t, *customConfigDefaults.DisplaySettings.ExperimentalTimezone, *ds.Get().DisplaySettings.ExperimentalTimezone)
```

Comment:
@mgdelacroix I'm seeing that some of these tests date back to https://github.com/mattermost/mattermost/pull/16265, but I didn't even know we had a way to provide custom defaults for the config. Should we add something to replace this setting for these tests, or do you think testing if the Site URL is set based on the custom config is enough?

Reply:
I feel that now that this setting is no longer use at all, it might be enough to only look at the URL and remove the other test entirely. Do you agree?